### ISSUE TITLE
WS7200 Doesn't work 

### ISSUE THREAD

**Issue by sklavosit** [vmakeev/huawei_mesh_router#17]
I try to connect the WS7200 AX3 QUAD CORE
I receive this error message: 
Retrying setup: 'wifi_80211r_switch'

**Comment by sklavosit** [comment]
After some minutes  I receive this message: Retrying setup: Authentication failed due to api call error; reason: auth_general

**Comment by vmakeev** [comment]
Hello!

I need a little more information to diagnose the problem.

Can you try installing the [latest release](https://github.com/vmakeev/huawei_mesh_router/releases/latest), enable debug logging, restart HA and try again?

Changes in `configuration.yaml`:

```
logger:
  .....
  logs:
    custom_components.huawei_mesh_router: debug
```

You may need to click the "load full logs" button to see the debug messages

Please attach component component debug logs to this discussion. They do not contain private data (passwords, usernames, external IP addresses)

-----

For successful authorization, the Home Assistant must be running on a device that is connected to the internal network of the router, via lan port, or via wifi, and not via the guest network.

**Comment by sklavosit** [comment]
This error originated from a custom integration.

Logger: custom_components.huawei_mesh_router.update_coordinator
Source: custom_components/huawei_mesh_router/update_coordinator.py:311
Integration: Huawei Mesh Router (documentation, issues)
First occurred: 5:57:57 PM (2 occurrences)
Last logged: 6:01:44 PM

Unexpected error fetching Huawei Mesh data: 'wifi_80211r_switch'
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 205, in _async_refresh
    self.data = await self._async_update_data()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 164, in _async_update_data
    return await self.update_method()
  File "/config/custom_components/huawei_mesh_router/update_coordinator.py", line 258, in async_update
    await self._update_switches()
  File "/config/custom_components/huawei_mesh_router/update_coordinator.py", line 311, in _update_switches
    self._switch_states[SWITCH_WIFI_80211R],
KeyError: 'wifi_80211r_switch'

**Comment by vmakeev** [comment]
It looks like Fast Roaming is not supported by your device, or there are some authentication issues. Debug logs (not just the error message) would be very helpful as I can't reproduce the error on my hardware.

However, one fix I have added, can you try installing the component from [this branch](https://github.com/vmakeev/huawei_mesh_router/tree/bugfix/unavailable_switch_logging)?

**Comment by vmakeev** [comment]
@sklavosit did you manage to check the updated version of the component? The update is also available in the [release v0.7.5.2](https://github.com/vmakeev/huawei_mesh_router/releases/tag/v0.7.5.2)

**Comment by sklavosit** [comment]
It works with  this release thank you
